test/all/run_all.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, output, errors):
 def run_command(cmdline):
     try:
         cmdline = " ".join(cmdline)
+        print(cmdline)
```

Comment:
This probably shouldn't be here.

Reply:
This is actually there on purpose so that you can directly reproduce a failed test run.